Source/items.h

```diff
@@ -242,6 +242,13 @@ struct Item {
 	uint32_t dwBuff = 0;
 	uint32_t _iDamAcFlags = 0;
 
+	friend Item Move(Item &source)

```

Comment:
Perhaps this could just be a method?

Reply:
That'd work, declaring a friend method was based on recommendations for defining generic swap functions (so ADL can find the specialisation) but because I used the DevX code style of PascalCase it wouldn't actually be useful in that way :D.

How about calling it Pop() to sounds like STL container pop_front/pop_back methods? In my mind `DoThing(item.Pop())` is easier to guess at the intent than `DoThing(item.Move())`

Reply:
PascalCase is for globals, members should be camelCase :P

Reply:
Welp, got a few things to tidy up now :D